doc/nrf/gs_ins_windows.rst

```diff
@@ -265,6 +266,48 @@ To install those, open a |bash| in the ``ncs`` folder and enter the following co
 
 .. add_deps_end
 
+.. _installing_ses_win:
+
+.. installing_ses_start
+
+Installing |SES|
+****************
+
+You must install a special version of |SES| to be able to open and compile projects in the |NCS|.
+
+|SES| is free of charge for use with Nordic Semiconductor devices.
+
+To install |SES|, complete the following steps:
+
+1. Download the package for your operating system from the following links:
+
+    * `SEGGER Embedded Studio (Nordic Edition) - Windows x86`_
+    * `SEGGER Embedded Studio (Nordic Edition) - Windows x64`_
+    * `SEGGER Embedded Studio (Nordic Edition) - Mac OS x64`_
+    * `SEGGER Embedded Studio (Nordic Edition) - Linux x86`_
+    * `SEGGER Embedded Studio (Nordic Edition) - Linux x64`_
+
+#. Extract the downloaded package.
+#. Register and activate a free license.
+   Complete the following steps:
+
+    a. Open |SES|. 
```

Comment:
This is not straight-forward, and since it's the first time, we should be more explicit (for example, "Run the file :file:`bin/emStudio`." as below).